lib/cocoapods/target/build_settings.rb

```diff
@@ -564,6 +564,12 @@ def initialize(target, test_spec = nil)
           paths
         end
 
+        # @return [String]
+        define_build_settings_method :framework_header_search_paths, :memoized => true do
```

Comment:
nit: it's a single path

Reply:
Just to be sure: You'd rename this `framework_header_search_path` (singular), right?